styleguide/_themes/global/scss/_fonts.scss

```diff
@@ -55,6 +55,9 @@ a {
     }
 }
 
+ul li, ol li{
+  text-align: left;
```

Comment:
Not in zoolander, in www. I'm not sure what is centering it, I haven't had time to delve.